backend/pom.xml

```diff
@@ -83,7 +83,7 @@
         <dependency>
             <groupId>eu.clarin.switchboard</groupId>
             <artifactId>profiler</artifactId>
-            <version>0.1.0-SNAPSHOT</version>
+            <version>1.0.0</version>
```

Comment:
I think a reference to this needs to be updated elsewhere. See failed build at https://travis-ci.org/github/clarin-eric/switchboard/builds/672018585#L1023